editor/plugins/game_view_plugin.cpp

```diff
@@ -845,6 +860,14 @@ void GameView::_update_arguments_for_instance(int p_idx, List<String> &r_argumen
 	List<String>::Element *N = r_arguments.insert_before(user_args_element, "--wid");
 	N = r_arguments.insert_after(N, itos(DisplayServer::get_singleton()->window_get_native_handle(DisplayServer::WINDOW_HANDLE, get_window()->get_window_id())));
 
+#if MACOS_ENABLED
+	r_arguments.push_back("--display-driver");
+	r_arguments.push_back("embedded");
+	// TODO(sgc): can we support compatibility?
+	r_arguments.push_back("--rendering-driver");
+	r_arguments.push_back("metal");
```

Comment:
At the moment, only Metal / Vulkan could be supported and this should specify Vulkan for `x86_64`. There is a way to render to a CAOpenGLLayer, which I'm going to explore as I might be able to support compatibility also.

Reply:
Not sure about this, maybe it's better to check renderer and disable embedding if it's GL for now. Silently switching a running project to a different renderer is not what most users will expect.

Reply:
No longer a problem, as I got OpenGL working too!